tools/flash-shelly.py

```diff
@@ -644,24 +640,65 @@ def device_scan(hosts, action, dry_run, quiet_run, silent_run, mode, exclude, ve
   parser.add_argument('-y', '--assume-yes', action="store_true", dest='silent_run', default=False, help="Do not ask any confirmation to perform the flash.")
   parser.add_argument('-V', '--version',type=str, action="store", dest="version", default=False, help="Force a particular version.")
   parser.add_argument('-c', '--hap-setup-code', action="store", dest="hap_setup_code", default=False, help="Configure HomeKit setup code, after flashing.")
+  parser.add_argument('-v', '--verbose', action="store", dest="verbose", choices=['0', '1', '2', '3', '4', '5'], default='3', help="Enable verbose logging 0=critical, 1=error, 2=warning, 3=info, 4=debug, 5=trace.")
   parser.add_argument('--variant', action="store", dest="variant", default=False, help="Prerelease variant name.")
-  parser.add_argument('-v', '--verbose', action="store", dest="verbose", choices=['0', '1'], help="Enable verbose logging level.")
+  parser.add_argument('--log-file', action="store", dest="log_filename", default=False, help="Create output log file with chosen filename.")
   parser.add_argument('hosts', type=str, nargs='*')
   args = parser.parse_args()
   action = 'list' if args.list else 'flash'
   args.mode = 'stock' if args.mode == 'revert' else args.mode
   args.hap_setup_code = f"{args.hap_setup_code[:3]}-{args.hap_setup_code[3:-3]}-{args.hap_setup_code[5:]}" if args.hap_setup_code and '-' not in args.hap_setup_code else args.hap_setup_code
-  if args.verbose and '0' in args.verbose:
-    logger.setLevel(logging.DEBUG)
-  elif args.verbose and '1' in args.verbose:
-    logger.setLevel(logging.TRACE)
+  sh = logging.StreamHandler()
+  sh.setFormatter(logging.Formatter('%(message)s'))
+  if args.verbose == '0':
```

Comment:
this kind of mapping is best done with a map

Reply:
how do you mean ?

Reply:
like this https://github.com/mongoose-os-apps/shelly-homekit/pull/396/commits/0568bef53ef4e7cff8434cbbe6bb3b9bb247301f